### PR TITLE
Moise/qiskit aer gpu

### PR DESCRIPTION
**Pull request by MoiseRousseau**
Add configuration file for the qiskit_aer package compiled with CPU/GPU support.

### CODE REVIEW

File: config/qiskit-aer-gpu-0.9.1.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+PACKAGE="qiskit-aer"
+PACKAGE_SUFFIX="-gpu"
+# setup.py takes package name from QISKIT_AER_PACKAGE_NAME environment
+# variable, so:
+PRE_BUILD_COMMANDS="export QISKIT_AER_PACKAGE_NAME=qiskit-aer-gpu"
+PACKAGE_DOWNLOAD_ARGUMENT="$PACKAGE==$VERSION"
```
Comment (ccoulombe) [inline config/qiskit-aer-gpu-0.9.1.sh:6]: This is already the default, no need to specify it.

**Review by ccoulombe — CHANGES_REQUESTED** [review]

File: config/qiskit-aer-gpu-0.9.1.sh (hunk 1 above)
Comment (MoiseRousseau) [reply config/qiskit-aer-gpu-0.9.1.sh:6]: The default is indeed "$PACKAGE==$VERSION", but I redefine the PACKAGE variable, so I have to redefine it. If I delete this lilne, it tries to download qiskit-aer-gpu==0.9.1 which does not provide source distribution. Source distribution is in qiskit-aer==0.9.1.

File: config/qiskit-aer-gpu-0.9.1.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+MODULE_RUNTIME_DEPS="
+    symengine/0.8.1
+    cuda/11.4
+    flexiblas/3.0.4
+    openmpi/4.0.3
```
[the remark this answers is not in the document]
Comment (MoiseRousseau) [reply config/qiskit-aer-gpu-0.9.1.sh:11]: I loaded cmake/3.22.1

File: config/qiskit-aer-gpu-0.9.1.sh (hunk 1 above)
Comment (ccoulombe) [reply config/qiskit-aer-gpu-0.9.1.sh:6]: Ok

File: config/qiskit-aer-gpu-0.9.1.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+"
+PYTHON_IMPORT_NAME=qiskit.Aer
+PYTHON_DEPS="cmake"
+BDIST_WHEEL_ARGS="--
+    -DBLA_VENDOR=FlexiBLAS
+    -DAER_THRUST_BACKEND=CUDA
+    -DAER_CUDA_ARCH='7.0;7.5;8.0;8.6;8.6+PTX'
```
[the remark this answers is not in the document]
Comment (MoiseRousseau) [reply config/qiskit-aer-gpu-0.9.1.sh:18]: I added CUDA_ARCH 6.0. 8.6 and 8.6+PTX are for compatibility with newer GPUs.

File: config/qiskit-aer-gpu-0.9.1.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (MoiseRousseau) [reply config/qiskit-aer-gpu-0.9.1.sh:12]: True. I just pushed the changes.

**Review by ccoulombe — APPROVED** [review]
:+1:

**Pull request merged**